src/google/protobuf/compiler/objectivec/objectivec_nsobject_methods.h

```diff
@@ -0,0 +1,90 @@
+// NSObject methods
```

Comment:
This file needs to be listed in the root `Makefile.am`.

Reply:
Done